optaplanner-core/pom.xml

```diff
@@ -94,8 +94,102 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- Benchmarking -->
+    <dependency>
+      <groupId>org.openjdk.jmh</groupId>
+      <artifactId>jmh-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.openjdk.jmh</groupId>
+      <artifactId>jmh-generator-annprocess</artifactId>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
-
+  <profiles>
+    <profile>
+      <id>benchmark</id>
```

Comment:
How difficult it is to run the benchmark locally?

At the very least, let's please document it in a readme. Let's also think about how to simplify it to a single step, if possible.

Reply:
Added small paragraph to README about starting benchmarks locally.